app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,57 @@
+class OrdersController < ApplicationController
+    before_action :order_page_access, only: [:index]
+
+  def index
+    @merchant = session[:user_id]
+    @orders = Order.includes(:order_items).where(order_items: { user_id: @merchant } )
+    # @order_item = OrderItem.find(params[:id])
+
+    if params[:status] == "paid"
```

Comment:
Hi Kari -- I gave removing the logic a try, but then the filter stops working and orders with a complete status no longer show up. If we have time after refactoring some of the bigger issues, I will try and take another look at how to refactor. Thanks!